ddtrace/contrib/graphql/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""
+The graphql integration instruments graphql-core queries. Version 2.0 and above are fully
+supported.
+
+
+Enabling
+~~~~~~~~
+
+The graphql integration is enabled automatically when using
+:ref:`ddtrace-run <ddtracerun>` or :func:`patch_all() <ddtrace.patch_all>`.
+
+Or use :func:`patch() <ddtrace.patch>` to manually enable the integration::
+
+    from ddtrace import patch
+    patch(graphql=True)
+    import graphql
+    ....
+
+Global Configuration
+~~~~~~~~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.graphql["service"]
+
+   The service name reported by default for graphql instances.
+
+   This option can also be set with the ``DD_GRAPHQL_SERVICE`` environment
+   variable.
+
+   Default: ``"graphql"``
+
+
```

Comment:
```suggestion
   To disable ``graphql.resolve`` spans set ``DD_GRAPHQL_SERVICE`` to False
   
   Default: ``True``
```

I left this as a comment for now. As of now, users would need to opt out of generating `graphql.resolve` spans. I think it's best if we disable these spans by default. For complex graphql queries we could produce hundreds of resolver spans per request. This could be overwhelming for new users and make flamegraphs unreadable.